src/System.Security.Cryptography.ProtectedData/tests/ProtectedDataTests.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.ProtectedDataTests
+{
+    public static class ProtectedDataTests
+    {
+        [Fact]
+        public static void RoundTripWithoutEntropy()
+        {
+            byte[] plain = { 1, 2, 3 };
+            byte[] encrypted = ProtectedData.Protect(plain, null, DataProtectionScope.CurrentUser);
+            byte[] recovered = ProtectedData.Unprotect(encrypted, null, DataProtectionScope.CurrentUser);
+            Assert.Equal<byte>(plain, recovered);
+        }
+
+        [Fact]
+        public static void RoundTripWithEntropy()
+        {
+            byte[] plain = { 1, 2, 3 };
+            byte[] entropy = { 4, 5, 6 };
+            byte[] encrypted = ProtectedData.Protect(plain, entropy, DataProtectionScope.CurrentUser);
+            byte[] recovered = ProtectedData.Unprotect(encrypted, entropy, DataProtectionScope.CurrentUser);
+            Assert.Equal<byte>(plain, recovered);
+        }
+    }
```

Comment:
What about negative tests?  Namely, changing the entropy between protect and unprotect.